src/components/AppHeader/AppHeader.tsx

```diff
@@ -46,7 +47,7 @@ const AppHeader: FC<HeaderProps> = () => {
           onClick={openMenu}
         />
         <Menu menuButtonRef={menuButtonRef}>
-          <MenuLinks menuSections={betaMenuSections} />
+          <NewMenuSections menuSections={betaMenuSections} />
```

Comment:
Does NewMenuSection replace MenuLinks ? If it's not used anywhere else maybe this should just be MenuSections

Reply:
It does here but I think `MenuLinks` is still used in the hamburger menu for the non-beta pages